v2/bugsnag_test.go

```diff
@@ -98,7 +84,16 @@ func TestNotify(t *testing.T) {
 
 	md := MetaData{"test": {"password": "sneaky", "value": "able", "broken": complex(1, 2), "recurse": recurse}}
 	user := User{Id: "123", Name: "Conrad", Email: "me@cirw.in"}
-	config := generateSampleConfig(ts.URL)
+
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
Also, can this be abstracted to function so it can be used consistently, and in future tests?

Reply:
refactored